test/presubmit-components-google-cloud.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -ex
+# Copyright 2020 Kubeflow Pipelines contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source_root=$(pwd)
+
+cd "$source_root/components/google-cloud"
```

Comment:
I'd suggest moving this script to this folder and call it  presubmit.sh, moving related things together generally helps maintenance

Reply:
Done, just to be sure you wanted me to move the `test/presubmit-components-google-cloud.sh` script to `components/google-cloud/` folder? if this is not correct kindly advise.